views/items/index.pug

```diff
@@ -1,9 +1,18 @@
 extends ../layouts/authenticated-layout
 
-mixin display( nodes, depth )
+mixin display( nodes, depth, root )
```

Comment:
I don't understand the changes here that introduce `root` - it was intended that this work generically for any tree.  Please discuss with me :)